### DISCUSSION TITLE
grammars located in /root/.config/helix/runtime

### DISCUSSION THREAD

**Discussion by KDDFKUFEI** [helix-editor/helix#11191] [Troubleshooting]
After fetching and building the grammars using doas hx -g fetch/build for some reason the queries, themes,  and tutor where placed in HELIX_RUNTIME which is /usr/share/helix/runtime(which is correct i think), but the grammers showed up in /root/.config/helix/runtime/.

I am using gentoo linux and installed helix using portage.
helix version is 24.3.

**Comment by the-mikedavis** [comment]
If you are installing Helix with a package manager then you shouldn't need to run `hx -g fetch` and `hx -g build` - the package should take care of installing grammars for you.

`hx -g fetch` and `hx -g build` not respecting `HELIX_RUNTIME` is a bug but it's already covered by https://github.com/helix-editor/helix/issues/9565
